netlify/functions/leaderboard.ts

```diff
@@ -39,25 +40,19 @@ const getLeaderboardResults = async (
     .filter((contest) => !contestId || Number(contestId) === contest.contestid);
 
   // get findings, filtered by contest
-  const allFindings = (await csv().fromFile("_data/findings/findings.csv"))
-    .map((finding) => {
-      finding.awardUSD = parseFloat(finding.awardUSD);
-      finding.split = parseInt(finding.split, 10);
-      return finding;
-    })
-    .filter((finding) =>
-      filteredContests.some(
-        (contest) => contest.contestid.toString() === finding.contest
-      )
-    );
+  const allFindings = (await getApiFindingsData()).filter((finding) =>
+    filteredContests.some((contest) => contest.contestid === finding.contest)
+  );
 
   // get deduplicated handles from findings
   const allHandles = Array.from(
-    new Set(allFindings.map((finding) => finding.handle))
+    new Set(allFindings.map((finding: AwardFinding) => finding.handle))
   )
-    .map((handle) => getWardenInfo(handle))
-    .filter((handle) => handle.showOnLeaderboard !== false);
-
+    .map((handle: string) => getWardenInfo(handle))
```

Comment:
n/m the existing code was already doing this. let's focus on landing these changes and moving on to other API work, rather than optimizing older code right now :-)

Reply:
Yah iwas trying to change as little code in this repo as possible to keep from causing any issues.